examples/weighted-average-of-peer-incoming-rate.config

```diff
@@ -0,0 +1,36 @@
+[default]
+# 'default' is special, it is used if no other policy matches a channel
+strategy = static
+base_fee_msat = 0
+fee_ppm = 50_000
+min_fee_ppm_delta = 50
+max_htlc_proportional_slices = 6
+
+# Ignore inactive nodes
+[inactive-nodes]
+node.max_shared_channels_active = 0
+strategy = ignore
+
+# Use the weighted average of the peer node's channels inbound rate.
+# Example:
+#    - Let's suppose you have a channel with Peer XYZ.
+#    - Peer XYZ has 4 channels.
+#    - Those 4 channels are 1M, 2M, 3M, and 4M sats capacity.
+#    - The inbound fee rates for those channels are 100 ppm, 200 ppm,
+#      300 ppm, and 400 ppm.
+#    - The weighted average is the sum of fees multiplied by the
+#      channel's capacity. In this case, 300 ppm. Note, it's higher
+#      than the plain average, because more capacity is at the 400 ppm
+#      rate compared to the lower ppm rate.
+#
+# The inbound_weighted_average_fee_rate_premium_percent value adds a
+# multiple of the weighted average fee. For example, a value of 25
+# means that a 200 ppm weighted average will be set to 250 ppm.
+
+[peer-adjusted]
+strategy = match_peer_inbound_weighted_average
+base_fee_msat = 0
+min_fee_ppm_delta = 50
+max_htlc_proportional_slices = 6
```

Comment:
For clarity I think these three lines could be removed (just as the elaborate [default] section above)